src/state/selectors.ts

```diff
@@ -7,47 +7,36 @@ import type { ReduxState } from './reducer'
 import type { LpId, StakingId, UserStakingId } from './slices/opportunitiesSlice/types'
 import type { PubKey } from './slices/validatorDataSlice/validatorDataSlice'
 
-// List of all the params filter consumed with selectParamFromFilter
-type ParamFilter = {
+/**
+ * List of all the params filter consumed with selectParamFromFilter
+ *
+ * note - this **must** stay as Partial, it's the selectors consumers responsibility to check
```

Comment:
this is kind of the crux of this PR - all filters are now optional. this PR most definitely isn't exhaustive, but it's a good step in the right direction. it's way too painful to change the assets + market data "base" selectors in this PR, but we can get this in and incrementally make nearly everything `Partial<T>`